### PR TITLE
Alt enums

### PR DESCRIPTION
**Pull request by burrbull**

### CODE REVIEW

**Comment by burrbull** [comment]
cc @therealprof review please

File: Cargo.toml [hunk 1]
```diff
@@ -279,6 +279,7 @@ stm32f446 = ["stm32f4/stm32f446", "device-selected", "gpio-f446",
     "otg-fs",
     "otg-hs",
     "sai",
+    #"sdio",
```
Comment (therealprof) [inline Cargo.toml:282]: What's up with that?

**Review by therealprof — COMMENTED** [review]
Looks very nice. Have you done any code size comparisons before/after?

File: Cargo.toml (hunk 1 above)
Comment (burrbull) [reply Cargo.toml:282]: 446 has sdio, but its svd has several issues

**Comment by burrbull** [comment]
> Looks very nice. Have you done any code size comparisons before/after?

approximatelly the same. +-2%

sometimes slightly bigger, simetimes smaller

**Review by therealprof — APPROVED** [review]
LGTM, thanls for working on this.

bors r+

**Comment by burrbull** [comment]
bors retry

**Pull request merged**